patches/llvm/0002-Add-managarm-target.patch

```diff
@@ -68,6 +69,8 @@ index 70fac030b..f1f0c30c3 100644
 +      Builder.defineMacro("_REENTRANT");
 +    if (Opts.CPlusPlus)
 +      Builder.defineMacro("_GNU_SOURCE");
++    if (this->HasFloat128)
++      Builder.defineMacro("__FLOAT128__");
```

Comment:
Why is this done?

Reply:
The Linux toolchain does this too, and we set `HasFloat128` to true for some targets, I figured we should probably add this then.